CONVERTER-SERVICE/src/database/database.db.js

```diff
@@ -0,0 +1,38 @@
+/*
+@database.db.js Copyright (c) 2022 Jalasoft
+2643 Av Melchor Perez de Olguin Colquiri Sud, Cochabamba, Bolivia.
+Av. General Inofuentes esquina Calle 20,Edificio Union № 1376, La Paz, Bolivia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const mongoose = require('mongoose');
+
+// Represents a Mongo Database
+class Database {
+  
+  constructor(uri, database){
+    this._uri = uri;
+    this.database = database;
+  }
+
+  // Connects to the data base
+  async connectDatabase() {
+    try {
+      await mongoose.connect( this._uri + this.database, {
+          useNewUrlParser: true,
+          useUnifiedTopology: true,
+      });
+    } catch (error) {
+        console.error();( 'Error: ', error );
```

Comment:
In this case you're already handling the error in the throw, is not necessary to console.error.

Reply:
It's already done